Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.shocksupershaman.shocktask1;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class UserDTO {

```

Comment:
You should keep the id. If I have a list of users I need to know their 'id' to call user specific operations for example updateUserNameById